### PR TITLE
Add instruction for install Feast on IKS and OpenShift using Kustomize

### PR DESCRIPTION
**Pull request by tedhtchang**
Signed-off-by: ted chang <htchang@us.ibm.com>

Add instruction for install Feast on IKS and OpenShift on IBM Cloud using Kustomize.
Remove instruction for install Feast on IKS with helm
Update urls in README, CONTRIBUTOR, and SUMMARY md files.

**Does this PR introduce a user-facing change?**:
NONE

### CODE REVIEW

File: docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md [hunk 1]
```diff
@@ -79,39 +84,43 @@ Therefore, to deploy Feast we need to set up [IBM Cloud Block Storage](https://c
    ```text
     ibmc-block-gold (default)   ibm.io/ibmc-block   65s
    ```
+### Security Context Constrain Setup
 
-   **3. Preparation**
-
-Add the Feast Helm repository and download the latest charts:
+By default, in OpenShift, all pods or containers will use the [Restricted SCC](https://docs.openshift.com/container-platform/4.6/authentication/managing-security-context-constraints.html) which prevents some pods to run as certain UID and causes installation to fail. In order to grant additional permissions, beyond those acquired with the restricted SCC, we need to use a different SCC. Since pods are created under the default or specified service accounts under the namespace, adding them to the `anyuid` SCC will resolve the failure. To do so, execute the following:
```
Comment (pvaneck) [inline docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md:89]:
```suggestion
By default, in OpenShift, all pods or containers will use the [Restricted SCC](https://docs.openshift.com/container-platform/4.6/authentication/managing-security-context-constraints.html) which limits the UIDs pods can run with, causing the Feast installation to fail. To overcome this, you can allow Feast pods to run with any UID by executing the following:
```

File: docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md [hunk 2]
```diff
@@ -79,39 +84,43 @@ Therefore, to deploy Feast we need to set up [IBM Cloud Block Storage](https://c
...
+git clone https://github.com/kubeflow/manifests
+cd manifests/contrib/feast/
+kustomize build feast/base | kubectl apply -n feast -f -
 ```
+### Optional: Enable Feast Jupyter and Kafka
 
-## 4. Installation
-
-Install Feast using Helm. The pods may take a few minutes to initialize.
+You may optionally enable the Feast Jupyter which contains code examples to demonstrate Feast. Some examples require Kafka to stream real time features to the Feast online serving. To enable, edit the following properties in the `values.yaml` under the `manifests/contrib/feast` folder:
```
Comment (pvaneck) [inline docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md:105]: add "component" after Jupyter.

**Review by pvaneck — CHANGES_REQUESTED** [review]

File: docs/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md [hunk 3]
```diff
@@ -1,23 +1,28 @@
-# IBM Cloud Kubernetes Service \(IKS\) \(with Helm\)
```
[the remark this answers is not in the document]
Comment (tedhtchang) [reply docs/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md:1]: This one will be replaced by the ibm-cloud-iks-with-kustomize.md.

**Comment by woop** [comment]
@tedhtchang we're not reviewing anything here yet since it's a draft. Let us know if you want us to look at anything.

File: docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md [hunk 4]
```diff
@@ -79,39 +84,43 @@ Therefore, to deploy Feast we need to set up [IBM Cloud Block Storage](https://c
    ```text
     ibmc-block-gold (default)   ibm.io/ibmc-block   65s
    ```
+### Security Context Constraint Setup
```
Comment (animeshsingh) [inline docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md:87]:
```suggestion
### Security Context Constraint Setup (OpenShift only)
```

File: docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md [hunk 5]
```diff
@@ -79,39 +84,43 @@ Therefore, to deploy Feast we need to set up [IBM Cloud Block Storage](https://c
...
-```bash
-helm install feast-release feast-charts/feast
+Then regenerate the resource manifests and deploy:
+```
+make feast/base
+kustomize build feast/base | kubectl apply -n feast -f -
 ```
 
-## 5. Use Jupyter to connect to Feast
+## 4. Use Feast Jupyter to connect to Feast
```
Comment (animeshsingh) [inline docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md:117]:
```suggestion
## 4. Use Feast Jupyter Notebook Server to connect to Feast
```

**Comment by tedhtchang** [comment]
@woop This PR is ready to review. Please take a look. Thanks.

**Pull request merged**